<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete_Book</class>
 <widget class="QMainWindow" name="Delete_Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(8, 8, 8);
font-family: Times New Roman;
COLOR: WHITE

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="formFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="bookIdLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BOOK ID: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtbookid">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid white;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DELETE BOOK</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { 
	background-color: #33ff39;
	border: 2px;
	border-radius: 8px;
	FONT: 12PX;
	COLOR: BLACK;

					
 }

QToolButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);}</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { 
	background-color: #33ff39;
	border: 2px;
	border-radius: 8px;
	FONT: 12PX;
	COLOR: BLACK;

					
 }

QToolButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);}</string>
    </property>
    <property name="text">
     <string>RETURN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
